initfiles/bash/etc/init.d/hpcc_common.in

```diff
@@ -445,6 +435,30 @@ startCmd() {
       fi
     fi
 
+    limits=(
+      MIN_Hn_nofile="32768"
+      MIN_Hc_core="unlimited"
+      MIN_Hu_nproc="8192"
+      MIN_Hr_rtprio="4"
+      MIN_Hl_memlock="unlimited"
+      MIN_Hs_stack="512" )
```

Comment:
This is difficult, if many threads are created and the stack size is high its a waste of memory.  If its too low then problems occur.  I am not sure what is best and it may vary for different programs.  Lets think about this and check if the programs alter the thread stacksize any and what is the default if its not set.